miniconda/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
```

Comment:
Looks like that's an untrusted cert.  Here's my output:

```
Step 4 : RUN wget https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh
 ---> Running in 121b4cdd80d5
--2015-04-20 15:48:37--  https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh
Resolving repo.continuum.io (repo.continuum.io)... 50.16.230.147, 23.23.87.95, 50.17.203.223, ...
Connecting to repo.continuum.io (repo.continuum.io)|50.16.230.147|:443... connected.
ERROR: The certificate of `repo.continuum.io' is not trusted.
ERROR: The certificate of `repo.continuum.io' hasn't got a known issuer.
```

@tpowellcio @lilmatt @bkreider: not sure who was involved with setting up SSL on repo.continuum.io.  Any ideas on what's going on here?  For reference, this is debian 7.4.